src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -988,6 +988,16 @@ const TorrentsTable = new Class({
         };
 
         // name, category
+        this.columns['name'].compareRows = function(row1, row2) {
```

Comment:
Maybe we should just apply this to all columns by default via `column['compareRows']` in `newColumn()`?

Reply:
Thanks. But as I explained to Chocobo1, I think some fields such as path, category and tags, may not expect case-insensitivity. Some people may have particular usage here.